windowing.py

```diff
@@ -205,3 +205,84 @@ def cropped_np(np_sample, start, end):
     end_index = np.where(nanos == end)
     cropped = [np_sample[:,  start_index[0][0]:end_index[0][0] + 1]]
     return cropped
+
+
+def place_data(np_sample, data_key, data_label, dictionary, positive, window_size, stride):
```

Comment:
nit: The name data_key is confusing to me. Both, data_key and data_label are keys, aren't they?